src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/MediaArchiveRecordMapperImpl.java

```diff
@@ -257,7 +257,9 @@ private static String getCourthouse(MediaEntity media) {
 
     private static String getCourtroom(MediaEntity media) {
         String courtroom = null;
-        if (nonNull(media.getCourtroom())) {
+        if (CollectionUtils.isNotEmpty(media.getHearingList()) && nonNull(media.getHearingList().get(0).getCourtroom())) {
+            courtroom = media.getHearingList().get(0).getCourtroom().getName();
+        } else if (nonNull(media.getCourtroom())) {
```

Comment:
Q - ticket says that media courtroom is nullable so should this drop in to here?

Reply:
What it should do it try the hearing first as it should really always have a hearing but in case there isn't it does the null check for courtroom before attempting to get the courtroom name from the courtroom